aws_google_auth/__init__.py

```diff
@@ -277,32 +283,41 @@ def cli():
     parser.add_argument('-S', '--sp-id', default=SP_ID, help='Google SSO SP identifier ($GOOGLE_SP_ID)')
     parser.add_argument('-R', '--region', default=REGION, help='AWS region endpoint ($AWS_DEFAULT_REGION)')
     parser.add_argument('-d', '--duration', default=DURATION, help='Credential duration ($DURATION)')
+    parser.add_argument('-p', '--profile', default=PROFILE, help='AWS profile ($AWS_PROFILE)')
 
     args = parser.parse_args()
 
-    if args.username is None:
-        args.username = raw_input("Google username: ")
-
-    if args.idp_id is None or args.sp_id is None:
```

Comment:
Added `raw_input` lookups for these values (below) if not resolved from the stored config file.